### PR TITLE
[Refactor] Remove clusterName in StatementBase

### PR DESCRIPTION
**Pull request by imay**

### CODE REVIEW

**Comment by imay** [comment]
run all

**Comment by imay** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 74 / 134 (55.22%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/analysis/AdminShowReplicaStatusStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowLoadWarningsStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/CancelStreamLoad.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/catalog/Database.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AdminShowReplicaDistributionStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowLoadStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AdminRepairTableStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/GetStreamLoadState.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowFunctionsStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/master/MasterImpl.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/QueryDumpAction.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|6|00.00%|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadManager.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/analysis/DropFileStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowBackupStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CancelLoadStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AlterDatabaseRename.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowSmallFilesStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CancelBackupStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/SelectStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AdminCancelRepairTableStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowExportStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/common/proc/ComputeNodeProcDir.java|0|4|00.00%|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowRestoreStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/Analyzer.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowDeleteStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateFileStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowRoutineLoadTaskStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowTransactionStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowRoutineLoadStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/LoadAction.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/RecoverDbStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowDynamicPartitionStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/StorageTypeCheckAction.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CancelExportStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/BaseAction.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/system/SystemInfoService.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzerUtils.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/alter/Alter.java|4|5|80.00%|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|24|28|85.71%|
|:large_blue_circle:|com/starrocks/analysis/RevokeStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowAlterStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/http/rest/TableQueryPlanAction.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowDataStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/AdminStmtAnalyzer.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/http/rest/TableRowCountAction.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/SetUserPropertyStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/mysql/privilege/Auth.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/http/rest/TableSchemaAction.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowTabletStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/common/MetaUtils.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowStmtAnalyzer.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/analysis/GrantStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateRoleStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/mysql/MysqlProto.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/WorkGroupAnalyzer.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/AlterDatabaseQuotaStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowCreateDbStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowUserPropertyStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/PrivilegeStmtAnalyzer.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/DropRoleStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateUserStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowPartitionsStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzeStmtAnalyzer.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/DropDbStmt.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/cluster/ClusterNamespace.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateDbStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/UserIdentity.java|1|1|100.00%|

File: fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.tokens [hunk 1]
```diff
@@ -0,0 +1,704 @@
+CONCAT=1
+ADD=2
+ADMIN=3
+AFTER=4
+AGGREGATE=5
+ALL=6
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.tokens:6]: why add this file?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 74 / 134 (55.22%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/analysis/AdminShowReplicaStatusStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowLoadWarningsStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/CancelStreamLoad.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/catalog/Database.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AdminShowReplicaDistributionStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowLoadStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AdminRepairTableStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/GetStreamLoadState.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowFunctionsStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/master/MasterImpl.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/QueryDumpAction.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|6|00.00%|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadManager.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/mysql/privilege/UserProperty.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/analysis/DropFileStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowBackupStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CancelLoadStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AlterDatabaseRename.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowSmallFilesStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CancelBackupStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/SelectStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/AdminCancelRepairTableStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowExportStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/common/proc/ComputeNodeProcDir.java|0|4|00.00%|
|:large_blue_circle:|com/starrocks/persist/CreateTableInfo.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowRestoreStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/Analyzer.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowDeleteStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateFileStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowRoutineLoadTaskStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/FunctionName.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowTransactionStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowRoutineLoadStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/LoadAction.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/RecoverDbStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowDynamicPartitionStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/rest/StorageTypeCheckAction.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/analysis/CancelExportStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/http/BaseAction.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/catalog/InfoSchemaDb.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/system/SystemInfoService.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzerUtils.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/alter/Alter.java|4|5|80.00%|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|24|28|85.71%|
|:large_blue_circle:|com/starrocks/analysis/RevokeStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowAlterStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/http/rest/TableQueryPlanAction.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowDataStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/cluster/Cluster.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/AdminStmtAnalyzer.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/http/rest/TableRowCountAction.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/SetUserPropertyStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/mysql/privilege/Auth.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/http/rest/TableSchemaAction.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowTabletStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/common/MetaUtils.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowStmtAnalyzer.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/analysis/GrantStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/TablePattern.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateRoleStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/mysql/MysqlProto.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/WorkGroupAnalyzer.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/AlterDatabaseQuotaStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/TableName.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowCreateDbStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowUserPropertyStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/PrivilegeStmtAnalyzer.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/DropRoleStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateUserStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/ShowPartitionsStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzeStmtAnalyzer.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/DropDbStmt.java|2|2|100.00%|
|:large_blue_circle:|com/starrocks/cluster/ClusterNamespace.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/CreateDbStmt.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/analysis/UserIdentity.java|1|1|100.00%|

**Review by wyb — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**